plugin/sampling/strategyprovider/adaptive/aggregator.go

```diff
@@ -152,7 +152,7 @@ func (a *aggregator) HandleRootSpan(span *span_model.Span, logger *zap.Logger) {
 	}
 	samplerType, samplerParam := span.GetSamplerParams(logger)
 	if samplerType == span_model.SamplerTypeUnrecognized {
-		return
+		samplerType = span_model.SamplerTypeProbabilistic
```

Comment:
@yurishkuro what kind of a config do we want to add to perform this override?

Reply:
something like "do not check sampler tags"

Reply:
@yurishkuro should this config be exposed as part of the YAML file? or do we just want it to be internal?

Reply:
It should be user settable